cli/src/skimmer.rs

```diff
@@ -7,79 +7,85 @@ use std::io::Cursor;
 use std::path::Path;
 use std::process::Command;
 
-pub fn table_skim() {
-    let manifest_file_path = filesystem::get_manifest_filename();
-    let tmp_manifest_file_path = filesystem::get_tmp_manifest_filename();
+const ASCII_ART: &str = "<< EOF
+___=____
```

Comment:
This is really, really a nit @biomunky but the top of the whale is now 4 spaces too far to the left. Can you move it back? lol